### ISSUE TITLE
BSO: Enable thieves armband at hams when thieving, but disable triple effect for clues.

### ISSUE THREAD

**Issue by badgehunter** [oldschoolgg/oldschoolbot#3971]
## 🚀 Feature Proposal

As titles says, make this work like it currently works on vyres. it lowers failure rate on them, it triples the loot on them, except for the shards which still drops with armband.

But instead on Vyres, at Ham members and for clues. It would lower failure rate on hams, it would triple the loot on them like it does currently but disable 3x effect for easy clues, which are currently entirely disabled if you have armband.

This would be good since then you dont need to give armband to your friend if you are not iron to get easy clues from hams, and irons dont need to get this entirely again if they want 3x effect on something else after pickpocketing hams for easy clues.

Tldr: make thieves armband work at hams on clues just like it currently does at vyres on blood shards.

**Comment by gc** [comment]
Done.
